src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -90,24 +99,54 @@ def create_container(client,
     if command_line is not None:
         command = shlex.split(command_line)
 
+    azure_file_volume = None
+    if azure_file_volume_share_name is not None:
```

Comment:
Change to `if azure_file_volume_share_name:` . 
Reference: https://www.udacity.com/wiki/cs258/truthiness-in-python